docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+
+### Registering a Google IAP subscriber
+
+SubPlat needs to be able to associate a Google IAP subscription in Google's system to a specific FxA user. This is done by registering each Google IAP subscriber with the [`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname) endpoint.
+
+By design, we allow only one FxA user per Play Store [`token`](https://developer.android.com/google/play/billing#tokens), which is how Google uniquely identifies the user's device when the subscription was purchased.
+
+This endpoint can be used to register a new Google IAP subscriber or to migrate an existing Google IAP subscriber. For the latter, see [(Optional) Migrate existing Google IAP subscribers to SubPlat](#optional-migrating-existing-google-iap-subscribers-to-subplat).
+
+### (Optional) Migrating existing Google IAP subscribers to SubPlat
+
+This section only applies to products that currently have a direct integration with Google who need to migrate existing subscribers to SubPlat.
+
+This step can be done by a developer. Before proceeding, complete the steps under [Configuration](#configuration) and [SubPlat API Calls](#subplat-api-calls). See also the [Testing your integration](#testing-your-integration) section.
+
+To migrate existing subscribers to SubPlat, use the existing registration endpoint ([`/subscriptions/iap/play-token/${appName}`](/ecosystem-platform/api#tag/Subscriptions/operation/postOauthSubscriptionsIapPlaytokenAppname)) to register each existing user with SubPlat.
+
+:::note
+
+While SubPlat routes can handle thousands of requests per second, we recommend limiting requests to 10-20 per minute to start and potentially increase that if there are no issues. This is because, at the time of writing, the Google Play Developer API docs states a default rate limit threshold of 200,000 queries per day, and we make more than one request to Google for each SubPlat request.
+
+:::
+
+## Testing your integration
+
+:::caution
+Like many systems, Google's Sandbox environment does not necessarily exhibit behavior consistent with what is stated in their docs. When in doubt, directly test the scenario in question, potentially more than once, to understand what to expect.
+:::
+
+### End-to-end testing of SubPlat APIs
+
+After development and the steps above are complete (and importantly, before involving QA), it is recommended for the lead RP integration engineer to pair with the lead SubPlat integration engineer to test the following:
+
+- [Getting the current subscription status for a FxA user](#getting-the-current-subscription-status-for-a-fxa-user)
+- [Registering a Google IAP subscriber](#registering-a-google-iap-subscriber)
+- [Receiving a Sandbox notification in our stage environment](#send-real-time-developer-notifications-to-subplat)
```

Comment:
This section title is different for Google notifications, so consider changing the link alias text?

Reply:
Good call. Done!